packages/create-app/package.json

```diff
@@ -24,16 +24,15 @@
         "lint": "eslint src",
         "lint:fix": "eslint src tests --fix",
         "typecheck": "tsc -p tsconfig.json --noEmit",
-        "test": "mocha",
+        "test": "vitest --run",
         "test:clear-baseline": "node scripts/clear_baseline",
         "test:accept-baseline": "node scripts/accept_baseline",
-        "coverage": "c8 npm test",
+        "coverage": "vitest --run --coverage.enabled --coverage.thresholds.100",
         "build": "tsup --silent",
         "prepublishOnly": "npm run build"
     },
-    "mocha": {
-        "import": "tsx/esm",
-        "spec": "tests/**/*.spec.ts"
+    "coverage": {
```

Comment:
What does this `coverage` property in `package.json` do?

If it's needed for `vitest` code coverage, should it also be in `packages/core/package.json`?

Reply:
Leftover from trying to configure coverage threshold (ended up going with flag in script). Will remove.